src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -25,7 +25,8 @@ public enum ApiIds
             GetAccounts = 1010,
             GetAccountById = 1011,
             GetAccountsByUserFlow = 1012,
-            RemoveAccount = 1013
+            RemoveAccount = 1013,
+            AcquireTokenForManagedIdentity = 1014
```

Comment:
1. Please add a comment that we won't see these API IDs in HTTP telemetry tables, because we don't hit ESTS directly
2. I'm thinking that we should separate SAMI and UAMI here for the ITelemetryClient ?